Anchor.toml

```diff
@@ -20,5 +20,12 @@ url = "https://api.apr.dev"
 cluster = "localnet"
 wallet = "~/.config/solana/id.json"
 
+[test]
+startup_wait = 60000
+
+[[test.genesis]]
+address = "2JWqYTXG5yHSU78hjKb39YFx82whbK74v6sMqMG3TVBQ"
+program = "target/deploy/degen_pools.so"
+
 [scripts]
-test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/*.test.ts --parallel"
```

Comment:
Why did we remove `--parallel`? It makes the testing much faster

Reply:
I agree, earlier it took less than 5 seconds, now it takes ~30 seconds on my machine. Reason I put this in is that I noticed some tests were randomly failing. If you ran them `.only` they pass, but all together one or two random cases fail.

Turns out the cases that randomly fail were all checking the fields emitted from events. My theory is that when tests run in parallel, the event caught by the listener is sometimes one from another testcase.

There's probably a neat way of working around this, but I haven't figured it out yet. As a hack, I made the tests run sequentially, and that seemed to solve the problem. It's not ideal, but hopefully the cost (30 seconds) is bearable.

Can you try running the tests in parallel on your machine? I wanna be sure it's not just failing on my machine.